typings/bs58.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'bs58'
```

Comment:
Can you add type declarations for `encode` and `decode`?

Reply:
yes, sure. I tried to use the official typings but it does not comply with the current specifications ;)

Reply:
done!